.chloggen/zpagesextension_confighttp.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: confighttp
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use confighttp.ServerConfig as part of zpagesextension
+
+# One or more tracking issues or pull requests related to the change
+issues: [9368]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
+
+# Optional: The change log or logs in which this entry should be included.
+# e.g. '[user]' or '[user, api]'
+# Include 'user' if the change is relevant to end users.
+# Include 'api' if there is a change to a library API.
+# Default: '[user]'
+change_logs: []
```

Comment:
can you confirm this is only breaking for the api?

Reply:
This also changes the options for users to enter the endpoint and additional options, right?

Reply:
Maybe I can have 2 changelog entries:
-breaking the api
-enhancement for user

Reply:
Right but this would only be breaking for users that were configuring options in TCPAddr that aren't in ServerConfig which I think is just the dial options correct? If i was only using endpoint, nothing should change since the struct is squashed?

Reply:
yes same field. Users are untouched. API is changed.